build/helm/sifnode/templates/ebrelayer-deployment.yaml

```diff
@@ -0,0 +1,82 @@
+{{- if .Values.ebrelayer.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sifnode.fullname" . }}-ebrelayer
+  labels:
+    {{- include "sifnode.labels" . | nindent 4 }}-ebrelayer
+spec:
+{{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+{{- end }}
+  selector:
+    matchLabels:
+      {{- include "sifnode.selectorLabels" . | nindent 6 }}-ebrelayer
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "sifnode.selectorLabels" . | nindent 8 }}-ebrelayer
+    spec:
+      serviceAccountName: {{ include "sifnode.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.ebrelayer.image.repository }}:{{ .Values.ebrelayer.image.tag }}"
+          imagePullPolicy: {{ .Values.ebrelayer.image.pullPolicy }}
+          command: ["/bin/sh"]
+          args: ["-c", "ebrelayer init $NODE_HOST $ETHEREUM_WEBSOCKET_ADDRESS $ETHEREUM_CONTRACT_ADDRESS $MONIKER --chain-id $CHAINNET --keyring-backend file"]
```

Comment:
As per @utx0's earlier comments, lets also change the ENV var here to `$ETH_BRIDGE_REGISTRY_ADDRESS` instead of `$ETHEREUM_CONTRACT_ADDRESS`

Reply:
Also change `--keyring-backend file` to `--keyring-backend test`.